### ISSUE TITLE
CDC COVID-19 - Bug: Some links aren't explicit

### ISSUE THREAD

**Issue by AccessForAll** [a11ySea/testing#18]
## Issue Summary

There are two "View More" links on the page, so they aren't explicit for screen reader users who may listen to them out of context. They also will be difficult for speech recognition software users to invoke either of these links.

## Steps to reproduce

1. Go to https://www.cdc.gov/coronavirus/2019-ncov/index.html
2. Scroll down to a "View More" link (there are two)
3. Right click on each link and select "Inspect Element"
4. View the code and note that there's insufficient additional context for either "View More" link (currently, this text is in a title attribute)

### Expected behavior

If these links can't be given explicit, visible link text instead of "View More", they must have additional descriptive text. This text could be in an aria-label or hidden text, so that screen reader users will be able to distinguish between these links when they read them in a contextual list of links.

### Actual behavior

- There's no additional text for these links, except in a title attribute, which most screen reader users won't be able to hear. Speech recognition users will have to use mouse grid to select either of these links; this is a lengthy process which slows down their user experience.

### Current Code
```
                            <a href="/coronavirus/2019-ncov/community/index.html" class="cta-button" title="View more about keeping communities safe">
                                View More
                            </a>


                            <a href="/coronavirus/2019-ncov/whats-new-all.html" class="cta-button" title="View latest updates">
                                View More
                            </a>
```

### Suggested Code
```
                            <a href="/coronavirus/2019-ncov/community/index.html" class="cta-button" title="View more about keeping communities safe" aria-label="View more about keeping communities safe">
                                View More
                            </a>


                            <a href="/coronavirus/2019-ncov/whats-new-all.html" class="cta-button" title="View latest updates" aria-label="View latest updates">
                                View More
                            </a>
```

## Additional References

- WCAG 2.1: Link Purpose in Context: https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html
